src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/containerregistry/container_registry.py

```diff
@@ -0,0 +1,84 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+#pylint: skip-file
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Code generated by Microsoft (R) AutoRest Code Generator 0.16.0.0
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
```

Comment:
our convention so far has been to put autorest output into a folder like "mgmt_acr"